README.md

```diff
@@ -49,6 +49,23 @@ For a full list of Poetry cli options visit the official documentation [here](ht
 - `make dependency-tree` or `make d-tree` Displays a dependency tree.
 - `make fix-imports` Automatically format imports for all project python files.
 
+## Configuration
+
+To change Flask's default settings, supply the relevant environment variables with
+the FLASK_ prefix. For more details see [here](http://flask.pocoo.org/docs/1.0/config/#environment-and-debug-features)
+
+To supply search settings that need to be accessed globally, add them to `search/constants.py`.
+It is recommended that you do not commit anything to this file but instead supply your
+own version during deployment to safeguard sensitive information.
```

Comment:
not just sensitive information (like a credential) but also portability as one official Docker image can be used by many deployments

Reply:
Would you like me update the documentation in some way?

Reply:
Would go for `to safeguard image portability and sensitive information`.

Reply:
how about now?

Reply:
:+1: